contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -25,6 +25,7 @@ CREATE TABLE sys.babelfish_function_ext (
 	flag_values BIGINT,
 	create_date SYS.DATETIME NOT NULL,
 	modify_date SYS.DATETIME NOT NULL,
+  	definition sys.NTEXT,
```

Comment:
Oh I missed to point this before, we need to specify `DEFAULT NULL` here otherwise it will be different from upgraded server.